### DISCUSSION TITLE
Download location

### DISCUSSION THREAD

**Discussion by jpayne120983** [fast4x/RiMusic#1226] [General]
Just wondering in which folder the downloaded songs get saved to.

**Comment by OrangeZXZ** [comment]
> Just wondering in which folder the downloaded songs get saved to.

RiMusic downloads songs in the Android folder i think also after download you can't play the songs without the player

**Comment by fast4x** [comment]
> Just wondering in which folder the downloaded songs get saved to.

No folder, downloaded music is crypted by player and not playable externally.

**Reply by f-mc2** [reply]
Can I ask why is it so? 

The issue I see with this choice is that, if I have to change smartphone for some reason, even if I import a backup, I have to download everything again.

**Reply by fast4x** [reply]
Unfortunately backup of songs cached or downloaded isn't possible. You can sync all your playlists with a piped account, so you have all your playlists in all devices.
https://piped.video
